libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -444,8 +451,7 @@ private <T> T getJsonProperty(JsonNode jsonNode, String propertyName, Collection
 
     private IdentityClaim getIdentityClaim(VerifiableCredential vc)
             throws HttpResponseExceptionWithErrorBody, CredentialParseException {
-        // TODO: PYIC-3908 Don't reparse here
```

Comment:
I think these comments kinda still apply? We might need to link to a different ticket if we end up deferring

Reply:
Ah ok, then I need to suppress the sonarcloud warnings

Reply:
Oh I see... If it's easier to just get rid of them that's fine, maybe reference the methods on the ticket instead then?